cdc/processor/sourcemanager/engine/factory/factory.go

```diff
@@ -54,6 +55,8 @@ type SortEngineFactory struct {
 	pebbleConfig *config.DBConfig
 	dbs          []*pebble.DB
 	writeStalls  []writeStall
+
+	dbInitialized *atomic.Bool
```

Comment:
Please comment why it's necessary.

Reply:
just name it `initialized` should be ok.